addons/account/static/src/js/bank_statement.js

```diff
@@ -7,7 +7,7 @@ odoo.define('account.bank_statement', function(require) {
     var includeDict = {
         renderButtons: function () {
             this._super.apply(this, arguments);
-            if (this.modelName === "account.bank.statement") {
+            if (this.modelName === "account.bank.statement" && this.$buttons) {
```

Comment:
Hello :wave: I this case you can use hasButtons to check if there is any to render
```suggestion
            if (this.hasButtons && this.modelName === "account.bank.statement") {
```

Otherwise, looks good to me

Reply:
I tried that approach, found out that either the $buttons var has a value or not I always find hasButtons set as false

Reply:
Hello, could you please try again?
We have tested with hasButtons and it does seem to work as expected.
Thank you!